### DISCUSSION TITLE
Can I edit table cell inplace without using v-edit-dialog?

### DISCUSSION THREAD

**Discussion by princesinghpnjr** [vuetifyjs/vuetify#15806] [General]
Can I edit table cell inplace without any popup or using v-edit-dialog ?

`<template v-slot:[`item.Amount`]="props">
                <v-edit-dialog
                  :return-value.sync="props.item.Amount"
                  large
                  persistent
                  @save="save1"
                  @close="close"
                >
                  <div>{{ props.item.Amount }}</div>
                  <template v-slot:input>
                    <div class="mt-4 text-h6">
                      <h6>Lender</h6>
                    </div>
                    <v-text-field
                      v-model="props.item.Amount"
                      :rules="[max25chars]"
                      label="Edit"
                      single-line
                      counter
                      autofocus
                    ></v-text-field>
                  </template>
                </v-edit-dialog>
              </template>`

**Comment by lpfy** [comment]
I am using v-menu with [quill text editor](https://github.com/surmon-china/vue-quill-editor) as a customised component for table cell editing purposes.

```
<v-data-table
    id="tableXYZ"
    dense
    :headers="tblXYZHeader"
    :items="tblXYZData"
    item-key="tblXYZId"
    :items-per-page="-1"
    hide-default-footer
>
    <template v-slot:[`item.FIELD_A`]="props">
        <TextEditor 
            :item="props.item"
            :refId="props.item.tblXYZId"
            :fieldName="'FIELD_A'"
            :openEdit="openEdit"
            :cancelEdit="cancelEdit"
            :saveEdit="saveEdit"
        />
    </template>
</v-data-table>
```

In your customised component TextEditor, define v-menu for the actual text editor
